fastlane/lib/fastlane/actions/pem.rb

```diff
@@ -46,13 +46,11 @@ def self.available_options
         require 'pem'
         require 'pem/options'
 
-        unless @options
-          @options = PEM::Options.available_options
-          @options << FastlaneCore::ConfigItem.new(key: :new_profile,
-                                       description: "Block that is called if there is a new profile",
-                                       optional: true,
-                                       is_string: false)
-        end
+        @options = PEM::Options.available_options
```

Comment:
@ohayon Not that I think of 🤔 The `unless @options` 100% breaks when `pem` is run more than once. I think that @KrauseFx wrote this originally so maybe we can get his 👀 on it?

Reply:
Yeah that would be helpful I think. My concern was just that this would override whatever `@options` is already set to. But maybe thats nothing to worry about 😄

Reply:
Yeah, it seems like that shouldn't be cached :)

Reply:
Woot! Had to rebase (tests were broken on the branch I branched off of). Can I get a 👍 on this when its done? 🙃